### DISCUSSION TITLE
Shutter32 in dev. 12.5.0.3 seems to be broken ..

### DISCUSSION THREAD

**Discussion by Mark-Muc** [arendst/Tasmota#18820] [Q&A] [answered]
Hello . just to let you know!!
I have attached 2 switches to a Sonnoff Dual R3 acting like opening / closing switch
in the final 12.05.0 to works like the following:
```
close Switch1 -> shutter moves down
open Switch1 -> shutter stops
close Switch2 -> shutter moves up
open Switch2 -> shutter stops
```
that's like it should be!!

in dev. 12.5.03 

```
close Switch1 -> shutter moves down
open Switch1 -> shutter moves on until it's endpoint
close Switch2 -> shutter moves up
open Switch2 ->shutter moves on until it's endpoint
```
even if it tryto fix it with  a rule like: 
```
Rule1 
ON Switch1#State=0 DO ShutterStop1 ENDON
ON Switch2#State=0 DO ShutterStop1 ENDON
Rule1 1

```
it won't stop .. the console shows me that the command 
`ShutterStop1 `
will be sent if i open the switches , but it shutter won't stop 
if i enter `ShutterStop1 `in the console manual .. it will stop the shutter!!
This behavior is different in the versions ... the way like 12.5.0 acts, it should be 
Thx & Regards
Mark_Muc

**Comment by Sfre12** [comment]
Hey. Did you solve your problem?
I have a problem closed to your, using V13.1.0.2. 
I have two shutters on one ESP. The switches 1 to 4 are not working propably to open or close the shutters.
I did a workaround. I set SetOption114 to "1" and control the shutter by script. (can be done by rule also, I think)
I can send you an example of my script if you need!

**Reply by Mark-Muc** [reply, marked as answer]
it was fixed in a later version .. the problem didn't show anymore!!
Mark_muc
